ElevatorSim/src/elevatorsim/server/ReceiverSocket.java

```diff
@@ -71,8 +71,8 @@ private void service(DatagramPacket request) {
 			} else if (DatagramPacketUtils.isElevatorRequest(request)) {
 				response = parentServer.handleElevatorRequest(request);
 				
-			} else if (DatagramPacketUtils.isStatusRequest(request)) {
```

Comment:
Are we not sending StatusRequests anymore?

Reply:
I replaced statusRequest with stateChangeRequest. I could change the naming back if you want

Reply:
No its ok - just wondering